package.json

```diff
@@ -1,23 +1,68 @@
 {
+  "name": "headset",
+  "productName": "Headset",
   "version": "2.2.0",
+  "description": "Discover and collect music on YouTube",
+  "license": "MIT",
+  "author": "Daniel Ravina <hello@headsetapp.co>",
+  "homepage": "http://headsetapp.co",
+  "bugs": "https://github.com/headsetapp/headset-electron/issues",
+  "main": "src/index.js",
+  "private": true,
```

Comment:
this probably needs to be public

Reply:
It was just a temporary way to stop showing the warning that Headset doesn't have a repository field. I'll delete this and instead point to the GitHub repository